lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -57,6 +69,10 @@ def __init__(self, *args, **kwargs):
         self._clean_restart = kwargs['config']['clean_restart']
         if self.config.multi_tunnel_flag:
             self._ovs_multi_tunnel_init()
+        #Get SessionD Channel
+        self._sessiond_setinterface = kwargs['rpc_stubs']['sessiond_setinterface']
+        self.loop = kwargs['loop']
```

Comment:
define public variables first and private ones last in __init__

Reply:
ok.